outlier_detection_cifar10/outlier_detection_cifar10.ipynb

```diff
@@ -5,13 +5,32 @@
    "id": "8bfe047b",
    "metadata": {},
    "source": [
-    "# Detecting out of distribution images in Cifar10 using Annoy and AutoML\n",
-    "\n",
+    "# Detecting out of distribution images in Cifar10 using Annoy and AutoML"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc1aafcc",
+   "metadata": {},
+   "source": [
+    "{{ badge }}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0bd0e883",
+   "metadata": {},
+   "source": [
     "This example shows how to train a neural network image classifier via [AutoGluon AutoML](https://auto.gluon.ai/) and then leverage it for outlier detection with cleanlab. We also demonstrate how to perform outlier detection using feature embeddings with custom K-Nearest-Neighbor implementations beyond ``sklearn``. Here we demonstrate this with the efficient approximate neighbors library [Annoy](https://github.com/spotify/annoy), but a similar strategy can be used to leverage any other KNN library. \n",
     "\n",
-    "You should be able to use the code here to find outliers in almost any image dataset.\n",
-    "\n",
-    "\n",
+    "You should be able to use the code here to find outliers in almost any image dataset.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94e0d277",
```

Comment:
Do we want this kind of "noise" in our notebooks? We could also write a quick sanitizer to get rid of this kind of stuff, and stuff like the Python version and whatever VS code adds, and then add to the linter a rule to ensure this is not there.